src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -492,6 +492,21 @@ parseNumberBinder = NumberBinder <$> (sign <*> number)
          <|> (symbol' "+" >> return id)
          <|> return id
 
+parseTypedBinder :: TokenParser Binder
+parseTypedBinder = do
```

Comment:
Not sure about this, I don't know what `typeableBinders` should contain, but I suspect it's some set of binders like I have here.